mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping

```diff
@@ -1 +1,25 @@
 CLASS net/minecraft/class_556 net/minecraft/client/render/item/HeldItemRenderer
```

Comment:
Please map the Minecraft argument in constructor for the sake of completeness

Reply:
Will do